src/store/reducers/contact.reducer.ts

```diff
@@ -53,10 +56,23 @@ const contactReducer = (state: IContactState = initialState, action: ContactActi
         loading: false,
       };
     }
+    case UPDATE_CONTACT_SUCCESS: {
+      return {
+        ...state,
+        contacts: action.payload,
```

Comment:
missing the sort?

Reply:
now the assignment is incorrect. you are assigning a single contact to an array

Reply:
I don't like this about React. A compilation error can be hidden while the application seem to work. I will test again.

Reply:
fixed